core/src/main/java/io/questdb/griffin/SqlCodeGenerator.java

```diff
@@ -2484,7 +2504,8 @@ private RecordCursorFactory generateTableQuery(
                         readerTimestampIndex,
                         functionParser,
                         myMeta,
-                        executionContext
+                        executionContext,
+                        latestByHasOnlyOneColumn
```

Comment:
this assumption is incorrect and regresses using index

missing `latest by` clause must not impact usage of the index